docs/en_US/preferences.rst

```diff
@@ -583,10 +583,40 @@ If auth_source is not given, Internal authentication will be consider by default
 
 .. code-block:: bash
 
-    /path/to/python /path/to/setup.py set-prefs user1@gmail.com sqleditor:editor:comma_first=true
+    /path/to/python /path/to/setup.py set-prefs user1@gmail.com --pref-options sqleditor:editor:comma_first=true
 
     # To specify multiple preferences options
-    /path/to/python /path/to/setup.py set-prefs user1@gmail.com sqleditor:editor:comma_first=true sqleditor:keyboard_shortcuts:toggle_case=true
+    /path/to/python /path/to/setup.py set-prefs user1@gmail.com --pref-options sqleditor:editor:comma_first=true sqleditor:keyboard_shortcuts:toggle_case=true
```

Comment:
This has been broken. multiple preferences are not getting saved.

Reply:
Done